src/index.ts

```diff
@@ -50,9 +58,20 @@ async function run() {
         ).stdout.includes('filter: lfs');
 
         if (shouldBeStoredInLFS) {
-          const isStoredInLFS = Boolean(
-            file.patch?.includes('version https://git-lfs.github.com/spec/v1')
-          );
+          let isStoredInLFS = false;
+          if (file.patch) {
```

Comment:
is it possible that `patch` would be _falsy_ , but still present ?
Should it be a more explicit check like 

```suggestion
          if (file.patch === undefined) {
```

?

Reply:
hmm I wanted to contemplate `null` & `undefined` at same time just in case at some point GitHub API changes and returns `patch: null` when no patch shall be returned & `== undefined | null` was a bit verbose